tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -823,6 +823,12 @@ public function dataFileAsserts(): iterable
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/array-replace.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6889.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/simplexml.php');
+
+		if (PHP_VERSION_ID >= 80100) {
+			yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6904.php');
+		}
+
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6917.php');
```

Comment:
The bugs in both of these issues should be tested in CallMethodsRuleTest. Feel free to keep these type inference tests, but duplicate the data files with a different namespace (for example Bug6904Rule) and test them in CallMethodsRuleTest.php to see if they're clean. Thanks!

Reply:
Done, thanks!